ui/client/stylesheets/visualization.styl

```diff
@@ -39,7 +39,11 @@ tipsPanel(bkrd-color, txt-color = undefined)
   tipsPanel(#4D4D4D)
 
 .tipsPanelHighlighted
-  tipsPanel(hrColor, black)
+  tipsPanel(alpha(errorColor,0.1), defaultTextColor)
```

Comment:
We have there some coding standard mismatch.. camelcase and separeted by `-`.. Whichone is better.. ? :)